.github/workflows/workspace_snippet.sh

```diff
@@ -7,6 +7,7 @@ set -o errexit -o nounset -o pipefail
 TAG=${GITHUB_REF_NAME}
 PREFIX="rules_deno-${TAG:1}"
 SHA=$(git archive --format=tar --prefix=${PREFIX}/ ${TAG} | gzip | shasum -a 256 | awk '{print $1}')
+LATEST_VERSION="$(bazel run //deno/private:latest_version --ui_event_filters=-info,-stdout,-stderr --noshow_progress)"
```

Comment:
just let the user load the LATEST_VERSION symbol from the same place they load deno_register_toolchains

Reply:
Good call, very clean!